RTN-006.tex

```diff
@@ -383,15 +384,15 @@ \subsubsection{Proactive Strategies for Diversity, Equity, and Inclusion}\label{
 Future versions of this model for community science will feature more fully developed descriptions of these proactive, DEI-driven strategies.
 
 
-\subsubsection{User Acceptance Testing of the RSP}\label{sssec:mod_dev_uat}
+\subsubsection{Science Validation of the RSP}\label{sssec:mod_dev_sv}
 
-The CST will conduct User Acceptance Testing (UAT) of the RSP and associated community-facing services at regular intervals, such as prior to major updates to the RSP, or before an annual Data Release.
+The CST will conduct Science Validation of the RSP and associated community-facing services at regular intervals. Examples include prior to major updates to the RSP or before an annual Data Release. The goal is to create, test, and execute a process to scientifically validate the RSP. This process should be designed to answer questions about the RSP's functionality and usability. Specifically, the validation process will aim to determine whether the RSP meets the scientific needs of its users, if the necessary tools exist, if users are able to perform scientific analyses in the RSP, and if the interface is inclusive of all users and potential users while being intuitive to use with minimal instruction. An understanding the diversity of Rubin users and RSP use-cases is key to developing the process for science validation of the RSP.
```

Comment:
content is good, just edit to follow the one-sentence-per-line convention which facilitates future changes